baseplate/__init__.py

```diff
@@ -18,13 +18,13 @@ def metrics_client_from_config(raw_config):
     This expects two configuration options:
 
     ``metrics.namespace``
-        The root key to namespace all metrics in this application under.
+        The root key to prefix all metrics in this application with.
     ``metrics.endpoint``
         A ``host:port`` pair, e.g. ``localhost:2014``. If an empty string, a
         client that discards all metrics will be returned.
 
-    :param dict raw_config: The app configuration which should have settings
-        for the metrics client.
+    :param dict raw_config: The application configuration which should have
```

Comment:
There are quite a few parameters documented similarly ("The application configuration which should have <X>") so I'll just be lazy and comment in one place for all.

I'm not 100% clear on how separate application configuration and settings for the metrics client (or the other <X>'s) are. I am interpreting this as "The application configuration, including settings for the metrics client".

Are the settings for the metrics client part of the application configuration, or are we combining separate things into one dict? Ditto for the other comments that are similar to this one.